cloudinary-core/src/main/java/com/cloudinary/metadata/SetMetadataField.java

```diff
@@ -0,0 +1,12 @@
+package com.cloudinary.metadata;
+
+import java.util.List;
+
+/**
+ * Represents a metadata field with 'Set' type.
+ */
+public class SetMetadataField extends AbstractMetadataValuesField<List<String>> {
```

Comment:
Problematic name...

Reply:
The word 'Set' is per the spec and API, I'm open to suggestions.